File: pytools/data/data.code.lua.client/ReqChatGm.lua
module("ReqChatGm", package.seeall)


function new()
	local tab_obj = {}
	setmetatable(tab_obj, {__index = ReqChatGm})
	tab_obj:init()
	return tab_obj
end

function init(self)
	self.content = 0
end


function encode(self)
	pack = packet.new()
	pack:writeString(self.content)
	return pack:encode(Msg.P_REQ_CHAT_GM)
end


function setContent(self, content)
	self.content = content
end

File: pytools/data/data.code.golang.client/req_chat_gm.go
package proto

import (
	"packet"
)

type ReqChatGm struct {
	content                  string
}

func (this *ReqChatGm) Encode() []byte {
	pack := packet.NewWriteBuff(64)

	pack.WriteString(this.content)

	return pack.Encode(P_REQ_CHAT_GM)
}

func (this *ReqChatGm) SetContent(content string) {
	this.content = content
}
